### ISSUE TITLE
Missing softmax layer

### ISSUE THREAD

**Issue by nihirc** [NVDLI/LDL#5]
In c5e2_mnist_learning_conf5 notebook for PyTorch, it's missing softmax layer. There is ReLU layer but missing softmax layer. 

Thanks!

**Comment by magnus-ekman** [comment]
In the Jupyter notebook I had included the following description for why the softmax activation is missing

> In this example we want to use ReLU as activation for the first layer and softmax for the second layer. One thing that is slightly unintuitive is that we omit the activation function altogether for the output layer. The reason is that the cross-entropy loss function implementation in PyTorch is implemented using the inputs to the softmax (also known as logits) instead of the outputs. This makes for an implementation that is more numerically stable (also see some more details in the section "Computer implementation of the cross-entropy loss function" in Chapter 5 in the book).

And the code is the following:
>model = nn.Sequential(
>    nn.Flatten(),
>    nn.Linear(784, 25),
>    nn.ReLU(),
>    nn.Linear(25, 10)
>)

So the layer nn.Linear(25, 10) in combination with the PyTorch implementation of the cross-entropy loss function will act as a softmax layer. Please let me know if this is still unclear or if I'm missing something.
